__tests__/app.test.js

```diff
@@ -135,3 +134,41 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("returns 200 status code and an array of comments for a valid article_id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .then((response) => {
+        const comments = response.body.comments;
+        expect(response.status).toBe(200);
+        expect(comments.length).toBe(11);
+        comments.forEach((comment) => {
+          expect(comment).toHaveProperty("comment_id");
+          expect(comment).toHaveProperty("body");
+          expect(comment).toHaveProperty("article_id");
+          expect(comment).toHaveProperty("author");
+          expect(comment).toHaveProperty("votes");
+          expect(comment).toHaveProperty("created_at");
+        });
+      });
```

Comment:
We would like to check the values too. Have a look into 
```
expect.objectContaining({
                body: expect.any(String),
```
We also want to check that the article_id matches the one we asked for :)